Add docs about `child()` method (#5)
jsumners/abstract-logging: Readme.md

All methods are no operation functions.

> Some loggers like Pino implement a `child()` method, which can be easily added to an `abstract-logging` instance in case it is needed:
> 

Reviewer: ```suggestion
Some loggers, like [Pino](https://getpino.io/), implement a `child()` method. This method can be easily added to an `abstract-logging` instance when stubbing out such loggers:

```
